index.js

```diff
@@ -231,35 +229,27 @@ plugin.uploadImage = function (data, callback) {
 		}
 
 		fs.readFile(image.path, function (err, buffer) {
-			uploadToS3(image.name, err, buffer, callback);
+			var maxWidth = parseInt(meta.config.maximumImageWidth);
```

Comment:
I think "normal" files and images are processed in different hooks and therefore in different handler methods.

Normal files are processed by ```plugin.uploadFile```